learn/getting_started/quick_start.md

```diff
@@ -269,6 +269,24 @@ If you have multiple indexes, you can switch between them using the indexes drop
 
 ![Meilisearch's search preview indicating the indexes dropdown in the upper right corner](/search_preview/multiple_indexes.png)
 
+## Customization
+
+At this point, you can configure your Meilisearch instance, and customize your index and search responses.  
+
+### Instance options
+
+These allow you to configure your Meilisearch instance at launch with [environment variables](/learn/configuration/instance_options.md#environment-variables) and [command-line options](/learn/configuration/instance_options.md#command-line-options-and-flags). You need to relaunch your instance to alter them.
+
+Instance options **affect your entire Meilisearch instance**, not just a single index.
```

Comment:
Minor, but since the subject is right next to the highlighted sentence, might as well include it and make it a fully-fledged clause.

```suggestion
**Instance options affect your entire Meilisearch instance**, not just a single index.
```